README.md

```diff
@@ -27,6 +27,8 @@ By default every local `<img src="image.png">` is required (`require('./image.pn
 
 You can specify which tag-attribute combination should be processed by this loader via the query parameter `attrs`. Pass an array or a space-separated list of `<tag>:<attribute>` combinations. (Default: `attrs=img:src`)
 
+If you use `<custom-elements>`, and lots of them make use of a `custom-src` attribute, you don't have to specify each combination `<tag>:<attribute>`: just specify an empty tag like `attrs=:custom-src` and it will match every element.
+
```

Comment:
@michael-ciniawsky Test added! You think we should add this config example right here? Or maybe what I added in the README should be better explained? It would be just having `attrs=:whatever-attribute` as stated above...